README.md

```diff
@@ -31,4 +31,4 @@ If you're unsure where to start, we curate a list of [good first issues](https:/
 If you have questions, you can contact us via IRC at [##zuzakistan-lab on freenode](irc://chat.freenode.net/##zuzakistan-lab)
 ([webchat](https://kiwiirc.com/nextclient/irc.freenode.net/##zuzakistan-lab)).
 
-If you encounter unacceptable behaviour in the community, please email conduct@zuzakistan.com.
+If you encounter unacceptable behaviour in the community, please email conduct@zuzakistan.com.
```

Comment:
I think this is here because you've changed whether this file ends with a newline character or not.

Reply:
It's the same line number though?

Reply:
<img width="769" alt="image" src="https://user-images.githubusercontent.com/817054/72434655-0feb7c80-3794-11ea-9ad2-fc6dad591313.png">

You've removed the invisible newline character at the end. [Blame POSIX.](https://thoughtbot.com/blog/no-newline-at-end-of-file)

You can remove this file from your commit via the command line (I assume there's a similar thing in the UI):
```bash
git reset HEAD^ README.md
git commit --amend
git push origin +master
```

Reply:
Had to add an extra commit - let me just fix that